pkg/apis/controlplane/v1beta1/conversion.go

```diff
@@ -380,3 +382,28 @@ func Convert_controlplane_AppliedToGroupPatch_To_v1beta1_AppliedToGroupPatch(in
 	out.RemovedGroupMembers = removedMembers
 	return nil
 }
+
+func Convert_controlplane_NetworkPolicy_To_v1beta1_NetworkPolicy(in *controlplane.NetworkPolicy, out *NetworkPolicy, s conversion.Scope) error {
+	out.ObjectMeta = in.ObjectMeta
+	out.Rules = *(*[]NetworkPolicyRule)(unsafe.Pointer(&in.Rules))
```

Comment:
don't we need to call `Convert_controlplane_NetworkPolicyRule_To_v1beta1_NetworkPolicyRule` here instead?

Reply:
Yes.... thanks for the catch.